public/js/view.js

```diff
@@ -281,38 +282,34 @@ function view(){
 				// Determine if there are references for the current gene
 				// AND its current mutation type
 				if (mutationIndex !== -1){
-					// Find all the cancers for which this gene is known to be mutated. Then add
-					// references for each row
-					var cancerToRefs = data.annotations[geneName][annotatedMutationNames[mutationIndex]],
-						cancerNames  = Object.keys(cancerToRefs).sort(),
-						refTable = [[
-								{type: 'text', text: 'Cancer'},
-								{type: 'text', text: 'PMID'},
-								{type: 'text', text: 'Edit'}
-							].map(gd3.tooltip.datum)
-						];
-
-					cancerNames.forEach(function(cancer){
-					    var refsUsed = [];
-						cancerToRefs[cancer].forEach(function(ref, i){
-							// only show the cancer name in the first row
-						    if (refsUsed.indexOf(ref.pmid) == -1) {
-							refsUsed.push(ref.pmid);
-							refTable.push([
-								{ type: 'text', text: i ? "" : cancerToName(cancer) },
-								{ type: 'link', body: ref.pmid, href: pubmedLink(ref.pmid)},
-							        { type: 'link', body: 'Edit', href: "/annotation/mutation/" + ref._id}
-							].map(gd3.tooltip.datum));
-						    }
-						});
-					});
+
+				    // Find all the cancers for which this gene is known to be mutated. Then add
+				    // references for each row
+				    var cancerToRefs = data.annotations[geneName][annotatedMutationNames[mutationIndex]],
+				    cancerNames  = Object.keys(cancerToRefs).sort();
 
 					// The table is hidden on default, so we show a string describing the
 					// table before showing it.
-					var knownAberrations = cancerNames.map(cancerToName).join(", ");
-					tooltipData.push({ type: 'text', text: 'Known ' + mutationClass + ' in ' + knownAberrations});
-					tooltipData.push({type: 'table', table: refTable, defaultHidden: true});
+				    var knownAberrations = cancerNames.map(cancerToName).filter(function(s) {return s != 'Unknown';}).join(", ");
+				    var inKnownCancers = knownAberrations == '' ? '' : (' in ' + knownAberrations);
+				    tooltipData.push({ type: 'text', text: 'Known ' + mutationClass + inKnownCancers});
+				    // count the number of references
+				    if (cancerNames) {
+					var numRefs = 0;
+					cancerNames.forEach(function(name) {
```

Comment:
Yes, it does store 'Unknown', but also as null, which is what the django database uses to represent unknown.  cancerToRefs maps cancers to references, and annotations are attached (this happens in the routes/view.js code)